<launch>
  <node pkg="hba" type="visualize_map" name="visualize_map" output="screen">
    <param name="file_path" type="string"
      value="/mnt/e/dataset/HBA/kitti07/"/>
    <param name="downsample_size" type="double" value="0.1"/>
    <param name="pcd_name_fill_num" type="int" value="5"/> <!--set 5 for kitti07 and 0 for park-->
    <param name="marker_size" type="double" value="0.5"/>
  </node>

  <!-- <arg name="rviz" default="1"/>
  <group if="$(arg rviz)">
    <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz"
    args="-d $(find hba)/rviz_cfg/rosbag.rviz" />
  </group> -->
</launch>